src/connections.c

```diff
@@ -1868,7 +1868,6 @@ connection_handle_read_post_state (request_st * const r)
             is_closed = 1;
     }
     else if (con->is_readable > 0) {
-        con->read_idle_ts = log_monotonic_secs;
```

Comment:
At first glance, moving this appears incorrect since the connection has been marked readable, and should have the last read timestamp updated when lighttpd attempts to read from the network.

The issue your reported might have a different, better solution to react to a `CLOSE_NOTIFY` TLS alert.

Reply:
agree, but is there any quick fix for this issue?

Reply:
agree, but is there any quick fix for this issue?

Reply:
haha.  You're using an ancient, outdated, unsupported, end-of-lifed openssl 1.0.2 and you're asking for a "quick fix"?

Please try to answer the questions I previously posted.

Reply:
the version is llighttpd 1.4.65,  and this problem is related with ssl3 which is eanbled by defaut on centos7.